### ISSUE TITLE
CRT Guest Advanced NTSC shader settings are displaying incorrectly

### ISSUE THREAD

**Issue by zltbugs** [libretro/slang-shaders#508]
When attempting to change the connection type within the CRT Guest Advanced NTSC shader settings, the options displayed are less than what is showing in the help.

Within the ntsc-pass1.slang file, the line that lists the Values, Svideo, Composite, RF does not have those options listed in the shader parameters within Retroarch.

Also there are long dashes that are also listed, while other sections are listed as [SECTION NAME].

`/shaders/shaders_slang/crt/shaders/guest/advanced/ntsc/ntsc-pass1.slang

#pragma parameter ntsc-row1 "------------------------------------------------" 0.0 0.0 1.0 1.0
pragma parameter quality "Values: Svideo = 0.0 | Composite = 1.0 | RF = 2.0" 0.0 0.0 1.0 1.0
pragma parameter cust_fringing "NTSC Fringing Value" 1.0 0.0 5.0 0.1
pragma parameter cust_artifacting "NTSC Artifacting Value" 1.0 0.0 5.0 0.1
pragma parameter ntsc-row2 "------------------------------------------------" 0.0 0.0 1.0 1.0`

**Comment by hizzlekizzle** [comment]
>less than what is showing in the help

Which 'help' are you referring to here?

**Comment by zltbugs** [comment]
> Which 'help' are you referring to here?

The scrolling text that displays the options for each setting.  Will post a screen capture when I have an opportunity to get back on that machine.

**Comment by zltbugs** [comment]
Added a screen shot of the issue.

**Comment by hizzlekizzle** [comment]
I think that was just an oversight on guest.r's part. That is, I suspect he change the values without changing the help text during one of the updates.

**Comment by nfp0** [comment]
That looks like a mistake indeed. Unless he removed one of the modes, which I doubt.

**Comment by nfp0** [comment]
I've noticed the `pragma parameter quality "Values: Svideo = 0.0 | Composite = 1.0 | RF = 2.0" 0.0 0.0 1.0 1.0` setting has no effect at all.
It worked fine before.

**Comment by hizzlekizzle** [comment]
Correct. It seems guest.r completely removed the quality presets. Dunno why.

**Comment by nfp0** [comment]
@guestrr Oh, I see what you mean now! Thanks for the input.
I need to catch up with the newer developments on your thread. :slightly_smiling_face:
